test/shapirowilk.jl

```diff
@@ -0,0 +1,116 @@
+using HypothesisTests, LinearAlgebra, Test, Random
+using StableRNGs
+
+@testset "ShapiroWilk" begin
```

Comment:
Can you add tests for lines not covered at https://app.codecov.io/github/JuliaStats/HypothesisTests.jl/blob/ShapiroWilk/src%2Fshapirowilk.jl?

(Not sure why coverage isn't reported in CI status.)

Reply:
probably because this pull is sooooo old;

while amending one of the first commits vscode told me it was changed 6(!) years ago...

Reply:
Probably.

There's still one uncovered line at https://app.codecov.io/github/JuliaStats/HypothesisTests.jl/blob/ShapiroWilk/src%2Fshapiro_wilk.jl. Do you know how to test it?